src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
-export interface Figure {
+export type Shape = 'triangle' | 'circle' | 'rectangle';
+export type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
 }
 
-export class Triangle {
+const getRounded = (num: number): number => {
+  return Math.floor(num * 100) / 100;
+};
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    if (
+      a <= 0
+      || b <= 0
+      || c <= 0
+      || c >= a + b
+      || b >= a + c
+      || a >= b + c
+    ) {
+      throw new Error('Enter correct data');
```

Comment:
it will be better to give more meaningful errors to let user understand what's wrong